packages/desktop-client/src/components/manager/subscribe/ChangePassword.tsx

```diff
@@ -60,7 +60,8 @@ export default function ChangePassword() {
         <Text
           style={{
             marginTop: 20,
-            color: colors.r4,
+            color: theme.errorText,
+            backgroundColor: theme.errorBackground,
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏I don't think we need this background color.

<img width="577" alt="Screenshot 2023-08-12 at 09 22 52" src="https://github.com/actualbudget/actual/assets/886567/3ca495f3-94c9-4220-8411-9ecb52bb584a">